pyproject.toml

```diff
@@ -112,18 +112,36 @@ exclude_lines = [
 [tool.tox]
 legacy_tox_ini = """
 [tox]
+envlist = lint,pytest
 isolated_build = True
-envlist = site
+labels =
+  venv = py311-lint, py3{6,7,8,9,10,11,12}-pytest
```

Comment:
Yay for testing on 3.12! \o/ Ideally we should remember to automatically add 3.13 etc.

Reply:
I don't think this is possible, unfortunately :(

Reply:
Spend a few minutes researching this and it seems that there indeed isn't a "py-latest".